src/pages/Artwork.js

```diff
@@ -8,12 +8,28 @@ const Artwork = () => {
   const [artwork, setArtwork] = useState(null);
   const { id } = useParams();
 
+  const historialStorage = useCallback( 
+    () => {
+      let array = [...JSON.parse(localStorage.getItem("historial") || "[]")];
+      const { title, images } = artwork;
+
+      if (array.some(elem => elem.id === id )) return null;
```

Comment:
Please add a comment here explaining this line, like "To avoid showing this same artwork as is pointless to navigate to the same artwork I am seeing"

Reply:
done